Describe Koa `ctx` elements available to Strapi (#1751)
strapi/documentation: docusaurus/docs/dev-docs/backend-customization/requests-responses.md

For more information, please refer to the [Koa response documentation](http://koajs.com/#response).

## CTX Table

### `ctx`
| Parameter | Description                                                                                  | Type     |
| ----------| -------------------------------------------------------------------------------------------- | -------- |

> | <a name="ctx" href="#ctx" class="anchor">`ctx`</a> | Ctx is the context object that is given to every layer within strapi         | `Object` |

Reviewer: ```suggestion
| <a name="ctx" href="#ctx" class="anchor">`ctx`</a> | Context object passed to every layer within Strapi         | `Object` |
```
